### PR TITLE
Add FluentPet source

### PR DESCRIPTION
**Pull request by theycantalkdatastudio**

### CODE REVIEW

**Comment by pfrisella** [comment]
Can you provide some detail on what type of data this represents?

**Comment by theycantalkdatastudio** [comment]
It allows users to export data gathered by https://fluent.pet/ mobile app. Info about who pressed the button, when, and in what kind of circumstances plus other metadata.

Columns exported by FluentPet's connector might include:
```
name,unique_id,event_type,is_human,species,breed,learner_language,birthdate,note,country,training_started_at,interaction_started_at,interaction_started_at_local,interaction_duration_in_seconds,teachers_in_household_count,learners_in_household_count,event_origin,board,all_data_json,buttons_count,all_buttons,all_contexts,button_1,button_1_concept,button_2,button_2_concept,button_3,button_3_concept,button_4,button_4_concept,button_5,button_5_concept,button_6,button_6_concept,button_7,button_7_concept,button_8,button_8_concept,button_9,button_9_concept,button_10,button_10_concept,button_11,button_11_concept,button_12,button_12_concept,button_13,button_13_concept,button_14,button_14_concept,button_15,button_15_concept,button_16,button_16_concept,button_17,button_17_concept,button_18,button_18_concept,button_19,button_19_concept,button_20,button_20_concept,context_1,context_2,context_3,context_4,context_5
```

image
image

File: sources/fluentpet.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "FLUENTPET",
+  "name": "FluentPet",
+  "categories": ["ANIMALS", "HARDWARE", "RESEARCH"],
```
Comment (pfrisella) [inline sources/fluentpet.json:4]: Remove ANIMALS category, it's too specific as a data category.
Can you use IOT instead of HARDWARE?

**Review by pfrisella — CHANGES_REQUESTED** [review]

**Review by pfrisella — APPROVED** [review]
LGTM. Thanks for adding!

**Pull request merged**